internal/Configuration.go

```diff
@@ -3,7 +3,9 @@ package internal
 import "github.com/caarlos0/env"
 
 type Configuration struct {
-	AppStoreAppVersionsSaveChunkSize int `env:"APP_STORE_APPLICATION_VERSIONS_SAVE_CHUNK_SIZE" envDefault:"20"`
+	AppStoreAppVersionsSaveChunkSize int    `env:"APP_STORE_APPLICATION_VERSIONS_SAVE_CHUNK_SIZE" envDefault:"20"`
+	ChartProviderId                  string `env:"CHART_PROVIDER_ID" envDefault:"*"`
```

Comment:
please add comments for chart provider id default value

Reply:
added